modules/acm/src/main/java/edu/gemini/epics/acm/CaObserveSenderImpl.java

```diff
@@ -56,6 +56,25 @@
     private ChannelListener<Short> stopMarkListener;
     private CaObserveSenderImpl.ApplyState currentState;
 
+    private ThreadFactory threadFactory = new ThreadFactory(){
```

Comment:
This looks familiar, would it be worth factoring out?

Reply:
👍